zone/raids.cpp

```diff
@@ -217,7 +225,7 @@ void Raid::RemoveMember(const char *characterName)
 	auto results = database.QueryDatabase(query);
 
 	Client *client = entity_list.GetClientByName(characterName);
-	Bot* bot;
+	Bot* bot = entity_list.GetBotByBotName(characterName);
```

Comment:
Did you mean to revert this? Could probably set default as `nullptr`.

Reply:
No.  Was an error in the push